3rd-party-tools/samtools-star-python/scripts/combined_mtx.py

```diff
@@ -0,0 +1,65 @@
+import argparse
```

Comment:
Do we still need this script since we no longer need to combine matrices? Should we just take it out?